InfiniTAM/ORUtils/Matrix.h

```diff
@@ -157,16 +163,16 @@ namespace ORUtils {
 		_CPU_AND_GPU_CODE_ inline Matrix4 &operator -= (const Matrix4 &mat) { for (int i = 0; i < 16; ++i) this->m[i] -= mat.m[i]; return *this; }
 
 		_CPU_AND_GPU_CODE_ inline friend bool operator == (const Matrix4 &lhs, const Matrix4 &rhs) {
-			bool r = lhs[0] == rhs[0];
+			bool r = lhs.m[0] == rhs.m[0];
```

Comment:
Were these changes motivated by your compiler having trouble with anonymous structs?

Reply:
This is not about anonymous struct. The Matrix<T> class simply doesn't provide a subscript operator. I think this part of codes were not tested before.